pkgs/development/python-modules/sqliteschema/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mbstrdecoder
+, tabledata
+, typepy
+}:
+
+buildPythonPackage rec {
+  pname = "sqliteschema";
+  version = "1.4.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
You also need to add setuptools to nativeBuildInputs. Also please change this for any other package in this PR.